kms_ext/extension.py

```diff
@@ -17,7 +18,7 @@
 from .secrets import EnvVar, Secret, SecretsFile
 
 log = structlog.get_logger()
-
+kms_key_id = os.environ.get("KMS_KEY_ID")
```

Comment:
The `get` method will return `None` if the key is not found, rather than raising a `KeyError`. Use `__getitem__` (i.e. `os.environ["KMS_KEY_ID"]`) instead to have it raise a `KeyError`.

```suggestion
try:
    kms_key_id = os.environ["KMS_KEY_ID"]
except KeyError as ex:
    raise Exception("The environment variable $KMS_KEY_ID must be set to decrypt") from ex
```

Since we only need the key within the decrypt method, and we wouldn't want to raise an exception otherwise, this code should be moved into the decrypt method, or into a function that is only called by the decrypt method.